zubhub_frontend/zubhub/src/views/project_details/ProjectDetails.jsx

```diff
@@ -491,12 +507,57 @@ function ProjectDetails(props) {
             })
           }
           aria-labelledby={t('projectDetails.ariaLabels.imageDialog')}
-        >
-          <img
-            className={classes.enlargedImageStyle}
-            src={enlarged_image_url}
-            alt={`${project.title}`}
-          />
+        > 
+         {/* Cancel Button on the Preview Images */}
+          <IconButton
+            className={classes.cancelEnlargedImageBtn}
+            onClick={() =>
+              setState({
+                ...state,
+                open_enlarged_image_dialog: !open_enlarged_image_dialog,
+              })
+            }
+            aria-label="close"
```

Comment:
aria labels should also be translated

Reply:
I have made note of all corrections, Ill make changes right away. Thank you @kamthamc